akka-http-core/src/main/scala/akka/http/impl/engine/server/HttpServerBluePrint.scala

```diff
@@ -295,11 +295,24 @@ private[http] object HttpServerBluePrint {
 
     override def onUpstreamFailure(error: Throwable, ctx: Context[ResponseRenderingContext]) =
       error match {
+        case EntityStreamSizeException(limit, contentLength) ⇒
+          val status = StatusCodes.RequestEntityTooLarge
+          val summary = contentLength match {
+            case Some(cl) ⇒ s"Request Content-Length of $cl bytes exceeds the configured limit of $limit bytes"
+            case None     ⇒ s"Aggregated data length of request entity exceeds the configured limit of $limit bytes"
+          }
+          val info = ErrorInfo(summary, "Consider increasing the value of akka.http.server.parsing.max-content-length")
```

Comment:
Yes, that's right. It is meant as a helper for people who are surprised by the error.
For these the comment should be helpful. If I have already applied a custom limit myself I probably know how to change things... at least that's my reasoning... :)